### PR TITLE
Throw error if browser name is not available on Sauce Labs

### PR DESCRIPTION
**Pull request by ralphtheninja**
New major.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ test('allows to retrieve all browsers on Sauce Labs', (t) => {
 });
 
 test('excludes browsers not available on Sauce Labs', (t) => {
```
Comment (lpinca) [inline test/test.js]: Can you please update the title?

File: index.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ function transform(wanted, available) {
   wanted.forEach((browser) => {
     const name = browser.name.toLowerCase();
 
-    if (!available.has(name)) return;
+    if (!available.has(name)) throw new Error(`Browser ${name} is not available`);
```
Comment (lpinca) [inline index.js]: Can you please keep 80 chars per line?

File: test/test.js [hunk 3]
```diff
@@ -24,8 +24,8 @@ test('allows to retrieve all browsers on Sauce Labs', (t) => {
...
-  sauceBrowsers([{ name: 'foo' }]).then((browsers) => {
-    t.deepEqual(browsers, []);
+  sauceBrowsers([{ name: 'foo' }]).catch((err) => {
+    t.equals(err.message, 'Browser foo is not available');
```
Comment (lpinca) [inline test/test.js]: Please also add `t.equal(err instanceof Error, true);` for consistency with the other tests.

**Review by lpinca — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (ralphtheninja) [reply index.js]: Sure, I wasn't sure about this since it's over 80 on other places.

Comment (ralphtheninja) [reply index.js]: Do you want `{}` when making new lines in if-statements?

Comment (lpinca) [reply index.js]: Yes, thanks, I think it will be consistent with the rest of the code base.

**Comment by lpinca** [comment]
Thank you.

**Comment by lpinca** [comment]
Published 2.0.0.

**Pull request merged**